brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapStore.swift

```diff
@@ -92,7 +92,7 @@ class SwapStore: NSObject, BaseDataStore, SwapDataStore {
         ?? supportedCurrencies?.first(where: { $0.baseCurrency == toCurrency?.code && $0.termCurrency == fromCurrency?.code })
     }
     
-    var minimumAmount: Decimal? {
-        return Decimal(pair?.minimumQuantity ?? 0)
-    }
+//    var minimumAmount: Decimal? {
+//        return quote?.minUsdAmount
+//    }
```

Comment:
goner?

Reply:
i commented it out, 2 see if smth will break.. .then when nothing did.. i forgot bout it xD